src/components/CortexScorecardWidget/CortexScorecardWidget.tsx

```diff
@@ -71,9 +130,10 @@ export const CortexScorecardWidget = () => {
 
   return (
     <EntityScorecardsCard
-      scores={scores}
+      title={'Cortex Scorecards'}
+      scores={scoresToDisplay}
       onSelect={scorecardId =>
-        navigate(`${location.pathname}cortex?scorecardId=${scorecardId}`)
+        navigate(`${location.pathname}/cortex?scorecardId=${scorecardId}`)
```

Comment:
could we handle both cases? like if there's a trailing `/` we remove it from `location.pathname`

Reply:
could you pull it down? this was working for me both when theres a trailing / and when there isnt

Reply:
ooh yes it works!